modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -519,10 +522,8 @@ class NDB_Form_Imaging_Browser extends NDB_Form
     function _updateVisitStatus()
     {
         if (!empty($_POST['visit_status'])) {
-            $save_visit_status
```

Comment:
I don't know, the code predates me.

I think 'Unrated' wasn't an option in the MySQL enum but was the value that was submitted by the HTML form when it was written for some reason, but I don't know of any good reason why. (It may have also been a bug that was introduced at some point that went uncaught for a long time, and as a result MySQL was changing the invalid enum option to `''`, and that became the "right" option to people looking into the database without the code being updated, but I don't know the history behind it. I suspect no one that remains does.)

I would keep the 'Unrated' check for now, because changing it would require changing historical data and removing the coalesce might stop the data from saving since it's not in the enum.